infrastructure/terraform/modules/templates-api/module_email_lambda.tf

```diff
@@ -0,0 +1,19 @@
+module "email_lambda" {
+  source      = "../lambda-function"
+  description = "templates api endpoint"
```

Comment:
I have a few thoughts on the structure here. Sorry for putting it all in one comment.

 I think largely around the continued use of `endpoint`. I only intended that lambda as like an example - the ticket I implemented it in (CCM-7249) asked for an empty backend, and I added the Lambda as an example of how you'd structure the code, build it, and deploy the Lambda via Terraform. But I think I intended it to be deleted and replaced with actual lambdas and endpoints - such as this email endpoint.

Can the `lambdas/endpoint` source code directory be renamed to something like `lambdas/email`.

And the `module.endpoint_build` become e.g. `module.email_lambda_build` 

The policy document `data.aws_iam_policy_document.endpoint_lambda_dynamo_access` should contain all of runtime permissions required by the specific lambda. It's not specific to Dynamo permissions and you're now adding SES permissions. So maybe just rename it to something like `data.aws_iam_policy_document.email_lambda`.

Reply:
updated the policies but kept the typescript package the same as discussed